libs/Einsums/TensorBase/include/Einsums/TensorBase/Common.hpp

```diff
@@ -14,6 +14,11 @@
 
 namespace einsums {
 
+/**
+ * @def DEFINE_STRUCT
+ *
+ * @brief Convenience macro for creating a type derived from a std::array.
+ */
 #define DEFINE_STRUCT(Name, UnderlyingType)                                                                                                \
```

Comment:
Since you expanded it out, is this even needed anymore?

Reply:
I don’t know. Might be useful, might not be. Do we plan on adding any more similar arrays?

Reply:
I don't know at the moment. Guess we can leave it for now.